### ISSUE TITLE
[4.0] Base Item not selectable in mod_menu

### ISSUE THREAD

**Issue by ReLater** [joomla/joomla-cms#19541]
### Steps to reproduce the issue
- Install J4.0 nightly build
- Open Module Manager
- Click New
- Select module of type "Menu"

- Field "Base Item" is empty. Nothing to select.

![02-02-_2018_20-23-08](https://user-images.githubusercontent.com/20780646/35751093-7412ecca-0857-11e8-8bee-a204b4491611.jpg)

Compare with Joomla 3.8

![02-02-_2018_20-26-18](https://user-images.githubusercontent.com/20780646/35751099-7a930774-0857-11e8-9d88-ce9cc3ed52bf.jpg)

**Comment by astridx** [comment]
I can confirm this issue.

**Comment by ReLater** [comment]
Looks like
```
 addfieldprefix="Joomla\Component\Menus\Administrator\Field"
```
is missing in mod_menu.xml

I didn't test if we need
```
 addfieldpath="/administrator/components/com_menus/models/fields"
```
any longer in mod_menu.xml.

**Comment by ghost** [comment]
closed as having Pull Request #19986
